pkg/system/path_windows.go

```diff
@@ -2,6 +2,26 @@
 
 package system
 
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
 // DefaultPathEnv is deliberately empty on Windows as the default path will be set by
 // the container. Docker has no context of what the default path should be.
 const DefaultPathEnv = ""
+
+// CheckSystemDriveAndRemoveDriveLetter verifies that a path, if it includes a drive letter,
+// is the system drive.
+func CheckSystemDriveAndRemoveDriveLetter(path *string) error {
```

Comment:
It's really unusual for go to pass strings pointers. Let's just return (string, error)